src/tests/ftest/ior/hard_rebuild.yaml

```diff
@@ -7,7 +7,7 @@ hosts:
     12_server:
       test_servers: server-[1-6]
   test_clients: 2
-timeout: 1000
+timeout: 1600
```

Comment:
it sounds weird that we increase the timeout and decrease the stone wall.
is the issue a performance regression?